test/pages/HasCommodityCodePageSpec.scala

```diff
@@ -28,4 +31,18 @@ class HasCommodityCodePageSpec extends PageBehaviours {
 
     beRemovable[Boolean](HasCommodityCodePage)
   }
+  "cleanup" - {
+    "should reset CommodityCodePage" - {
+      "when HasCommodityCodePage is changed to No" in {
+        val emptyUserAnswers = UserAnswers("id", DraftId(1))
+
+        val ua = emptyUserAnswers
+          .set(CommodityCodePage, "123456")
+          .get
+
+        HasCommodityCodePage.cleanup(Some(false), ua) mustBe
+          Success(emptyUserAnswers)
+      }
+    }
```

Comment:
We want to be testing all scenarios here, namely what happens when the cleanup is called with a None, and Some(true).